config/rbac/common/gateway_clusterrole.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
do we need `gateways` to be cluster wised? is it only for `istio-system` namespace?

Reply:
We list the gateways in all namespaces https://github.com/kyma-project/lifecycle-manager/blob/b2e06b62b67b29841bbbc8b11ec617125e4b7e12/internal/istio/client.go#L103

Reply:
OK, this we need to improve, not make sense to list all gateways, but we can do it in a followup issue.